### ISSUE TITLE
UpdateProfile doesn't include "options" parameter

### ISSUE THREAD

**Issue by BodiePaden** [klaviyo/klaviyo-api-node#83]
The Klaviyo docs say that UpdateProfile will return the "subscriptions" block, but it actually doesn't.  The only way I was able to get it was by adding `additional-fields[profile]=subscriptions` to the PATCH request in Postman.  There doesn't seem to be an equivalent in the SDK as the function takes no "options" parameter to support it.

I'm guessing this is the case on other functions as well like CreateProfile, etc., but I haven't tested those.

**Comment by nat-klaviyo** [comment]
Thanks for raising this, we'll investigate a fix for a future release. As a workaround for now, could you try making a GET request after your PATCH with `additionalFieldsProfile: ['subscriptions']`?

**Comment by BodiePaden** [comment]
Yes, that's exactly what I'm doing, thank you.  Looking forward to the fix so I don't have to make the extra call.  :)
